my-app/src/components/Medicines/Inputs.js

```diff
@@ -17,6 +19,11 @@ const ContactTableRow = props => {
         setData({ ...data, [name]: value });
     };
 
+    const { t } = useTranslation();
+  function handleClick(lang) {
```

Comment:
This is not needed nor used here. We can remove this function :)

Reply:
deleted